content/doc/book/pipeline/index.adoc

```diff
@@ -7,7 +7,8 @@ title: Pipeline
 :author:
 :email: jenkinsci-users@googlegroups.com
 :sectanchors:
-:imagesdir: /doc/book/resources
+ifdef::env-github[:imagesdir: resources]
+ifndef::env-github[:imagesdir: ../resources]
```

Comment:
@gilesgas I think this should be `:imagesdir: ../resources` in both cases.

Reply:
You're right @bitwiseman - I tested this change locally with a `make run` build and pushed to GitHub pages and this change works in both cases.